pservice/Makefile

```diff
@@ -62,14 +62,14 @@ $(CONTRACT_ENCLAVE_MRENCLAVE_C_FILE): $(CONTRACT_ENCLAVE_MRENCLAVE_TEMPLATE_FILE
 	VAR_MRENCLAVE=$$(perl -0777 -ne 'if (/metadata->enclave_css.body.enclave_hash.m:([a-fx0-9 \n]+)/) { $$eh = $$1; $$eh=~s/0x| |\n//g; $$eh=~tr/a-z/A-Z/; print "$${eh}\n"; }' $(CONTRACT_ENCLAVE_MRENCLAVE_META_FILE)) && \
 	sed "s/MR_ENCLAVE_PLACEMARK/$${VAR_MRENCLAVE}/" $(CONTRACT_ENCLAVE_MRENCLAVE_TEMPLATE_FILE) > $(CONTRACT_ENCLAVE_MRENCLAVE_C_FILE)
 
-$(ENCLAVE_LIB) : build $(ENCLAVE_FILES)
+$(ENCLAVE_LIB) : build
 	@echo Build Enclave
 	$(MAKE) -C build
```

Comment:
this is a simple fix... but doesn't it still leave open the problem of parallelism between eservice & pservice builds? how do you know the eservice build will complete first? there are some dependencies that *could* be captured by moving exclusively to CMAKE. What's your thinking about "bandaids" versus real solutions?

Reply:
> doesn't it still leave open the problem of parallelism between eservice & pservice builds? how do you know the eservice build will complete first?

First, `build.sh` delineates the overall build procedure, and the eservice and pservice builds are "serialized" in there.
Second, if we run the eservice/pservice builds in parallel and the eservice does not generate `eservice/build/lib/libpdo-enclave.signed.so.meta`, the pservice build will fail as it needs that.

> there are some dependencies that could be captured by moving exclusively to CMAKE

I don't disagree. However, that cmake file would likely cover both eservice and pservice, thus tightening their integration (as opposed to having them fully independent).

Perhaps a better solution is: implement a `get_mrenclave` function in the eservice, an `add_trusted_mrenclave` in the pservice, and transfer the mrenclave operationally (rather than at build time). In this way, the services will be fully independent and can be build in parallel.

> What's your thinking about "bandaids" versus real solutions?

If the real solution is the cmake one, I'm not sure about the advantages -- maybe you can elaborate further.
For now, this PR _fixes_ the build in HW mode.

Reply:
why not just have the eservice build a common include file at the end of building the enclave so? the pservice just includes the .h file with a definition in it. the basic idea of having the pservice build that file seems broken. the whole template thing is pretty obfuscated.

Reply:
And... we can fix the build simply by removing the extraneous line that dropped the multi-processing in the build. I'll add that fix to my PR (the first change in your PR).